src/panels/config/zha/zha-device-card.ts

```diff
@@ -224,13 +224,19 @@ class ZHADeviceCard extends LitElement {
             `
           )}
         </div>
-        <div class="card-actions">
-          <mwc-button @click=${this._addToLovelaceView}>
-            ${this.hass.localize(
-              "ui.panel.config.devices.entities.add_entities_lovelace"
-            )}
-          </mwc-button>
-        </div>
+        ${
+          this.device!.entities && this.device!.entities.length > 0
```

Comment:
```suggestion
          this.device?.entities?.length > 0
```

Reply:
this cooks CI

Reply:
had a quick chat with balloob and his explanation about the error makes sense. This could evaluate to undefined > 0. Should I just put it back? If so, we can close this PR and use #4541  which would have needed a rebase after this one but is good to go if we close this.